tests/system/fixtures/mom-ee-disabled-1.4.json

```diff
@@ -12,21 +12,28 @@
     ]
   ],
   "container": {
-    "type": "DOCKER",
+    "type": "MESOS",
     "docker": {
       "image": "mesosphere/marathon-dcos-ee:__FILL_IN_VERSION_HERE__",
-      "network": "HOST",
+      "pullConfig": {
+        "secret": "config-json"
+      },
       "privileged": false,
-      "parameters": [],
       "forcePullImage": false
-    }
+    },
+    "volumes": []
   },
   "env": {
     "JVM_OPTS": "-Xms256m -Xmx2g",
     "PLUGIN_ACS_URL": "https://master.mesos",
```

Comment:
in all other jsons I see "PLUGIN_ACS_URL": "http://master.mesos" - http instead of https. So maybe you forgot to change it here?